packages/web-components/src/wallet-connection/watchWallet.js

```diff
@@ -125,9 +128,33 @@ export const watchWallet = (chainStorageWatcher, address, rpc) => {
         updatePurses(brandToPurse);
       };
 
-      const watchBank = async () => {
-        const balances = await queryBankBalances(address, rpc);
-        bank = balances;
+      /** @type {Tendermint34Client} */
+      let tendermintClient;
+      const watchBank = async (numRetries = 0) => {
```

Comment:
"retries" requires subtracting one to know the number of attempts. I think this would be clearer,
```suggestion
      const watchBank = async (attempts = 0) => {
```

Reply:
Done